runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -6519,6 +6519,8 @@ TR::Register *J9::Power::TreeEvaluator::VMnewEvaluator(TR::Node *node, TR::CodeG
 
          static bool disableFastArrayZeroInit = (feGetEnv("TR_DisableFastArrayZeroInit") != NULL);
 
+         iCursor = generateTrg1ImmInstruction(cg, TR::InstOpCode::li, node, zeroReg, 0, iCursor);
+
```

Comment:
zeroReg is definitely set up before this point already.  read back some of the code please ...

Reply:
it is used before this ... i can see.

Reply:
I have to double check, but I found through both code walkthrough and debugging, the zeroReg value was altered to something else. I will post it later.